patch

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
This file is used when the PKGBUILD.source file and octave-pkgname.install file change. It can apply the changes to all the PKGBUILD file and .install file in the subdirectories.